### PR TITLE
Dami/chain extension

### PR DESCRIPTION
**Pull request by dharjeezy**

### CODE REVIEW

**Comment by drewstone** [comment]
@dharjeezy can we change the branch this work goes into to `develop`

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -39,6 +40,19 @@ use pallet_transaction_payment::CurrencyAdapter;
 #[cfg(any(feature = "std", test))]
 pub use sp_runtime::BuildStorage;
 pub use sp_runtime::{Perbill, Permill};
+/*use webb_primitives::{
```
Comment (drewstone) [inline runtime/src/lib.rs]: can we remove this if it is not needed?

**Review by drewstone — APPROVED** [review]
LGTM, just curious why we don't remove commented out codes.

File: runtime/src/lib.rs (hunk 1 above)
Comment (dharjeezy) [reply runtime/src/lib.rs]: I will remove them now...

**Pull request merged**